Integrations/EWSv2/EWSv2.py

```diff
@@ -103,6 +104,10 @@
 AUTO_DISCOVERY = False
 SERVER_BUILD = ""
 MARK_AS_READ = demisto.params().get('markAsRead', False)
+MAX_FETCH = int(demisto.params().get('maxFetch', 50))
```

Comment:
`demisto.params().get('maxFetch', 50)` can return `None`, which will cause exception (`TypeError: int() argument must be a string or a number, not 'NoneType'`) and is generally a bad idea (e.g. did you know that `None < 0` is true in python?)

To avoid this, split into two parts:
```
MAX_FETCH = demisto.params().get('maxFetch')
MAX_FETCH = MAX_FETCH if MAX_FETCH and 0 < MAX_FETCH <= 50 else 50
```

The bonus is you don't need the if statement in the next 2 lines.